test/conformance/service_test.go

```diff
@@ -20,15 +20,15 @@ package conformance
 
 import (
 	"fmt"
-	"strconv"
-	"testing"
-
 	"github.com/google/go-cmp/cmp"
 	pkgTest "github.com/knative/pkg/test"
 	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
 	"github.com/knative/serving/test"
+	"github.com/knative/serving/test/e2e"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strconv"
```

Comment:
nit: Put standard library packages at the top and separate them

https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Moved to the top along with `fmt` and `testing` packages